retail/retail-java-applications/terraform/README.MD

```diff
@@ -114,7 +120,7 @@ google_project_service.pubsub: Creating...
 .
 .
 google_pubsub_subscription.transactions_inbound_sub: Creation complete after 12s [id=projects/[your project ID]/subscriptions/Transactions-inbound-sub]
-Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
+Apply complete! Resources: 12 added, 0 changed, 0 destroyed.
 ```
 ## Terraform Destroy
```

Comment:
Running ``terraform destroy``` gives me this error: ```Error: cannot destroy instance without setting deletion_protection=false and running `terraform apply````

Reply:
- The problem was that an additional flag "deletion_protection" was introduced for big_query_table in [terraform-provider-google](https://github.com/hashicorp/terraform-provider-google/blob/master/CHANGELOG.md) 3.58.0 released on 23 Feb 2021 to prevent accidental deletion of BigQuery tables.
- Previously installed provider version 3.57.0 was reused when running `terraform init` because it satisfied the required provider version `>=3.48.0`.
- The problem was replicated by cloning a new repo and running terraform scripts from there which installed google provider version 3.58.0.
- Will change the required provider version to `>=3.58.0` instead and always test with latest version of provider.